docs/tutorials/tutorial_with_repositoryserver.rst

```diff
@@ -0,0 +1,539 @@
+      phases:
+      - func: BackupDataUsingKopiaServer
+        name: backupToS3
+        args:
+          namespace: "{{ .Deployment.Namespace }}"
+          pod: "{{ index .Deployment.Pods 0 }}"
+          container: test-container
+          includePath: /var/log
+    restore:
+      inputArtifactNames:
+      - timeLog
+      - backupIdentifier
+      phases:
+      - func: ScaleWorkload
+        name: shutdownPod
+        args:
+          namespace: "{{ .Deployment.Namespace }}"
+          name: "{{ .Deployment.Name }}"
+          kind: Deployment
+          replicas: 0
+      - func: RestoreDataUsingKopiaServer
+        name: restoreFromS3
+        args:
+          namespace: "{{ .Deployment.Namespace }}"
+          pod: "{{ index .Deployment.Pods 0 }}"
+          image: ghcr.io/kanisterio/kanister-tools:0.92.0
+          backupIdentifier: "{{ .ArtifactsIn.backupIdentifier.KeyValue.id }}"
+          restorePath: /var/log
+      - func: ScaleWorkload
+        name: bringupPod
+        args:
+          namespace: "{{ .Deployment.Namespace }}"
+          name: "{{ .Deployment.Name }}"
+          kind: Deployment
+          replicas: 1
+  EOF
+
+After creating a blueprint, the events associated with it can be viewed by using
+the following command:
+
+.. code-block:: yaml
+
+  $ kubectl --namespace kanister describe Blueprint time-log-bp
+  Events:
+    Type     Reason    Age   From                 Message
+    ----     ------    ----  ----                 -------
+    Normal   Added      4m   Kanister Controller  Added blueprint time-log-bp
+
+The next CustomResource to be deployed is an ActionSet. An ActionSet is created
+whenever there is a need to execute Kanister actions. It contains all
+the runtime information that is essential for the controller during execution.
+It can include multiple actions, each acting on a different Kubernetes object.
+In this tutorial, the forthcoming ActionSet specifies the ``time-logger``
+deployment that was previously created and selects the ``backup`` action
+within our Blueprint.
+
+
+Add some data in the time logger app.
+
+.. code-block:: bash
+
+   kubectl exec -it time-logger-6d89687cbb-bmdj8 -n default -it sh
+   sh-5.1# cd /var/log/
+   sh-5.1# ls
+   time.log
+   sh-5.1# echo "hello world" >> test.log
+   sh-5.1# cat test.log
+   hello world
+
+ActionSet
+---------
+
+.. code-block:: bash
+
+  # Create action set using the blueprint created in above step
+  $ kanctl create actionset --action backup --namespace kanister --blueprint time-log-bp --deployment default/time-logger --repository-server kanister/kopia-repo-server
+  actionset actionset backup-rlcnp created
+
+The ``--repository-server`` flag is used to provide the reference to the repository server
+CR created in step :ref:`Creating Repository Server custom resource <creating_repo_server_CR>`.
+As the CR contains details related to the Kopia repository server and the associated secrets,
```

Comment:
```suggestion
The ``--repository-server`` flag is used to provide a reference to the repository server
CR created in step :ref:`Creating Repository Server custom resource <creating_repo_server_CR>`.
Since the CR contains details related to the Kopia repository server and the associated secrets,
```